test/e2e/testnet/testnet.go

```diff
@@ -26,16 +26,21 @@ type Testnet struct {
 	txClients []*TxSim
 }
 
-func New(name string, seed int64, grafana *GrafanaInfo) (*Testnet, error) {
+func New(name string, seed int64, grafana *GrafanaInfo, chainID string,
+	genesisModifiers ...genesis.Modifier) (
+	*Testnet, error,
+) {
 	identifier := fmt.Sprintf("%s_%s", name, time.Now().Format("20060102_150405"))
 	if err := knuu.InitializeWithScope(identifier); err != nil {
 		return nil, err
 	}
 
+	gen := genesis.NewDefaultGenesis().WithChainID(chainID).WithModifiers(genesisModifiers...)
```

Comment:
[optional] instead of a strange variable name like `gen`, we could inline this in the only place it's used below:

```go
		genesis: genesis.NewDefaultGenesis().WithChainID(chainID).WithModifiers(genesisModifiers...),
```

Reply:
Sure, addressed https://github.com/celestiaorg/celestia-app/pull/3391/commits/5eea75475272900b57d9a1cba565c65953d0f4ee